### PR TITLE
[15.0][MIG] account_netting: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT36939

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
LGTM

File: account_netting/tests/test_account_netting.py [hunk 1]
```diff
@@ -5,140 +5,51 @@
 from datetime import datetime
 
 import odoo.tests.common as common
+from odoo.exceptions import ValidationError
 
 
-class TestAccountNetting(common.SavepointCase):
+class TestAccountNetting(common.TransactionCase):
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py:14]: missing post-install tag and remove at-install

File: account_netting/tests/test_account_netting.py [hunk 2]
```diff
@@ -5,140 +5,51 @@
...
-                        0,
-                        {
-                            "name": "Test",
-                            "price_unit": 100.0,
-                            "account_id": cls.account_revenue.id,
-                        },
-                    )
-                ],
-            }
+        cls.aa_model = cls.env["account.account"]
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py:25]: No need to create this stuff with the new approach.

File: account_netting/tests/test_account_netting.py [hunk 3]
```diff
@@ -148,88 +59,115 @@ def setUpClass(cls):
...
+            {
+                "name": "Test %s journal" % journal_type,
+                "type": journal_type,
+                "code": code,
+                "company_id": self.company.id,
+            }
+        )
+
+    def _create_move(self, move_type, journal, partner, account, price):
+        return self.env["account.move"].create(
```
Comment (pedrobaeza) [inline account_netting/tests/test_account_netting.py]: Use `Form`

**Review by pedrobaeza — COMMENTED** [review]
Put yourself as contributor

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**